docs/tutorials/ifci_px4_flight_controller.rst

```diff
@@ -0,0 +1,304 @@
+
+    Save the Configuration
+
+Once saved, build the firmware with the following command, replacing ``your-flight-control-board`` with the name of your flight control board. This will be the same name as used in the :ref:`previous steps<Setting Up the PX4 Toolchain>`.
+
+.. code-block:: bash
+
+    make your-flight-control-board
+
+Your firmware file should appear in the ``PX4-Autopilot/build/your-flight-control-board_default`` folder as ``your-flight-control-board_default.px4``
+
+.. warning::
+    Adding the IFCI configuration features will increase the size of the PX4 build. This could make the build larger than the flash size available on your board.
+    
+    .. figure:: ../_static/tutorial_images/ifci_px4_flight_controller/large_code.png
+        :align: center
+        :scale: 50
+        :alt: Oversized Code
+
+        Oversized Code
+
+    If the build is too large does, you will have to turn off other features to fit the binary on your flight controller. We recommend excluding other output drivers (PWM, DShot, DroneCAN, etc.) that you will not be using. Features can be turned off through boardconfig exactly as you turned on Vertiq’s features.
+
+    .. figure:: ../_static/tutorial_images/ifci_px4_flight_controller/disable_outputs.png
+        :align: center
+        :scale: 50
+        :alt: DShot and PWM Location
+
+        DShot and PWM Location
+
+    With the build settings updated, rebuild the program. When the compiled program occupies under 100% of the flash size, you will be able to fully build the PX4 application.
+
+    .. figure:: ../_static/tutorial_images/ifci_px4_flight_controller/small_code.png
+        :align: center
+        :scale: 50
+        :alt: Trimmed Code
+
+        Trimmed Code
+
+Flashing PX4 to Your Flight Controller
+======================================
+
+Once built, you must program your flight controller with your compiled ``.px4`` file. To do this, open QGroundControl, go to the vehicle settings menu and enter the 'Firmware' menu. Once there, plug in your board, select the 'Advanced Settings' checkbox, and then the 'Custom Firmware' option.
+
+.. figure:: ../_static/tutorial_images/ifci_px4_flight_controller/custom_firmware.png
+    :align: center
+    :scale: 50
+    :alt: Custom Firmware Selection
+
+    Custom Firmware Selection
+
+Once you press 'OK', a file explorer should appear. Find the ``your-flight-control-board_default.px4`` file that you built and select it. The flashing process should begin. 
+
+Enabling IFCI on Your Flight Controller
+===================================================
+
+Once the flashing is complete, connect to your flight controller with QGroundControl and go to the parameters menu. In the parameters menu :red:`search for 'vertiq'`. The parameter ``VERTIQ_IO_CFG`` :blue:`should appear`. Once you find this, :green:`set it to the serial port that you plan on using`, save, and reboot the flight controller as instructed by QGroundControl.
```

Comment:
Here's a scary question given how awful it is...but should we have a discussion in here about how serial ports map...? PX4's docs suckkkk

Reply:
I think for most standard board the names written on the board are the same as they'll find in QGC. This actually didn't come up as an issue with the two people who have tried it so far.